src/db/table/user_payment.sql

```diff
@@ -0,0 +1,13 @@
+-- User payment records all users' payments on the platform
+CREATE TABLE `user_payment` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `payment_type_id` int(11) DEFAULT NULL COMMENT 'Can be null if we decide to stop supporting a payment method',
+  `user_id` int(11) DEFAULT NULL,
+  `date` datetime NOT NULL,
+  PRIMARY KEY (`id`),
+  KEY `user_payment_payment_type_idx` (`payment_type_id`),
```

Comment:
nit: u2u to change this, but these should match the foreign key name with an _idx suffix. In the end, though, this won't make a difference, so feel free to ignore.